content/fringes/pulppluginday.md

```diff
@@ -4,3 +4,16 @@ date: 2018-11-23
 eventdate: "6th February 2019"
 draft: true
 ---
+Pulp is a platform for managing repositories of software packages, but it can store any kind of
+binary data (e.g. cat photos). The core provides a REST API and manages files, but the real value
+for users is delivered by the plugins which allow core to manage a content type like a cat photo.
+
+Please join some of the Pulp developers on February 6th for Pulp Plugin Day. This will be an
+opportunity to add new features to any of the
+[existing plugins](https://pulpproject.org/pulp-3-plugins/) or start on a new plugin to support
+another content type.
+
+We are going to start at 10 am by going over the
+[Plugin Writer's Guide](https://docs.pulpproject.org/en/3.0/nightly/plugins/index.html#plugin-writer-s-guide).
+Then everyone is going to setup a development environment and start working on a Pulp plugin of
+their choice. 
```

Comment:
It would be good to have a paragraph on the experience level. A typical question people have is if they have enough experience.

Reply:
what about an idea to invite users to email with questions or if they want to get involved but aren't sure how?

Reply:
Well, if you want to be more open, you should have something for the different levels, and try to have topics by difficulty so different people can join in
As for the mail, we want to centralize registration and add it to the general registration, we can add an open question to allow people to ask questions, and maybe ask them for their experience level and their python skills

Reply:
Well, if you want to be more open, you should have something for the different levels, and try to have topics by difficulty so different people can join in
As for the mail, we want to centralize registration and add it to the general registration, we can add an open question to allow people to ask questions, and maybe ask them for their experience level and their python skills

Reply:
@toshywoshy I like the idea of asking "Have you used Pulp before? Describe your experience with Python."

Reply:
They have been added to the registration form